### ISSUE TITLE
List prerequisites to run in GitHub docs

### ISSUE THREAD

**Issue by sjug** [code-ready/crc#215]
Trying to work with CRC on:
`Linux ryzen 5.1.7-arch1-1-custom #1 SMP PREEMPT Thu Jun 6 07:40:25 EDT 2019 x86_64 GNU/Linux`

Check `README.md` for instructions, instructions to read docs are to build docs first.
Clone repo, try to build docs:
```
[user@host crc]$ make build_docs
podman run -v /home/jugs/go/src/github.com/code-ready/crc/docs:/docs:Z --rm registry.gitlab.com/gbraad/asciidoctor-centos:latest -b html5 -D /docs/build /docs/source/getting-started/master.adoc
WARN[0000] cannot run `loginctl enable-linger` for the current user: exit status 1
cannot clone: Operation not permitted
user namespaces are not enabled in /proc/sys/kernel/unprivileged_userns_clone
Error: could not get runtime: cannot re-exec process
make: *** [Makefile:81: build_docs] Error 125
```

Turns out I need podman on a distro that supports rootless mode to even build the docs. (This should be publicly documented here)
Does it need to be rootless as not all distros have the latest shadow-utils?

I find the two steps I need to run elsewhere:
```
$ ./crc setup
$ ./crc start -b <downloaded_bundle_tarball>
```

Okay, lets try this:
```
[user@host crc]$ ./crc setup
crc - Local OpenShift 4.x cluster
INFO Caching oc binary
INFO Setting up virtualization
INFO Setting up KVM
INFO Installing Libvirt
INFO Adding user to libvirt group
INFO Enabling libvirt
INFO Starting Libvirt service
INFO Installing crc-driver-libvirt
[sudo] password for jugs:
INFO Setting up Libvirt crc network
INFO Starting Libvirt crc network
INFO Writing Network Manager config for crc
FATA Failed to write NetworkManager config file: /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf: tee: /etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf: No such file or directory
: exit status 1
```

Turns out I need NetworkManager installed as well. (This should be publicly documented here)

Please state the prerequisites publicly so users can install them before encountering errors.

**Comment by praveenkumar** [comment]
@sjug Thanks for trying out and filling the bug, we are in process to put a clear documentation about those prerequisites if someone not using  Fedora/RHEL/CentOS.  cc @robin-owen

**Comment by robin-owen** [comment]
I believe this issue was fixed by #277 some time ago. :)
